infra/base-images/base-builder/Dockerfile

```diff
@@ -82,6 +82,8 @@ ENV SANITIZER_FLAGS_dataflow "-fsanitize=dataflow"
 
 ENV SANITIZER_FLAGS_thread "-fsanitize=thread"
 
+ENV SANITIZER_FLAGS_introspector "-flto"
```

Comment:
When cloning LLVM you're extracting LLVM 14, and in this case we need to use `-flegacy-pass-manager -flto` for introspector